modules/local/methylkit/main.nf

```diff
@@ -0,0 +1,49 @@
+process METHYLKIT {
+    label "process_medium"
+
+    conda "${moduleDir}/environment.yml"
+    container "075615082992.dkr.ecr.us-west-2.amazonaws.com/methylkit:latest"
+
+    input:
+    val qualimap_output
+    path samplesheet
+    path bismark_cov_dir
+    path genome_assembly
```

Comment:
```suggestion
    val genome_assembly
```
This actually is supposed to get read in by the markdown as a string (we never open the fasta file just use the name of the fasta to determine whether to do certain human and mouse specific analyses)